app/assets/javascripts/darkswarm/services/current_order.js.coffee

```diff
@@ -1,4 +1,16 @@
 Darkswarm.factory 'CurrentOrder', (currentOrder) ->
   # Populate Currentorder.order from json in page. This is probably redundant now.
   new class CurrentOrder
-    order: currentOrder
+    # order: currentOrder
+
+    getOrder: ->
```

Comment:
Hi @bingxie. I don't really like the look of this. What is the problem that this function is solving? Why do we need state_id and country_id to be strings?